server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/ErrorMessageHelper.java

```diff
@@ -40,7 +40,7 @@ public class ErrorMessageHelper {
   private static final String MSG_PREFIX = "{";
   private static final String MSG_SUFFIX = "}";
 
-  ErrorMessageHelper(MessageService messageService) {
+  public ErrorMessageHelper(MessageService messageService) {
```

Comment:
I think changes in this file can be reverted.
```suggestion
  ErrorMessageHelper(MessageService messageService) {
```